eng/test-mounts.sh

```diff
@@ -31,3 +29,9 @@ for unit_path in "$content_snap_path"/mounts/*.mount; do
     [[ -d $unescaped_path ]] && echo "Directory $unescaped_path exists"
     [[ $(find "$unescaped_path" -maxdepth 1 | wc --lines) -gt 1 ]] && echo "Directory $unescaped_path is not empty"
 done
+
+# Test .NET output
+cp -Lp "$dotnet_path"/dotnet $mount_destination_path
+cp -RLp "$dotnet_path"/host $mount_destination_path
```

Comment:
```suggestion
cp --dereference --preserve=mode,ownership,timestamps "$dotnet_path"/dotnet "$mount_destination_path"
cp --recursive --dereference --preserve=mode,ownership,timestamps "$dotnet_path"/host "$mount_destination_path"
```

suggestion: use long flag names to improve readability

nitpick: `$mount_destination_path` is not put in `""`. Apart from a potential issue this may cause if the path ever will contain spaces; it is inconsistent with the "command" below/overall code style of the script.

Reply:
Done!